src/main/resources/static/js/subscription.js

```diff
@@ -175,7 +175,8 @@ jQuery(document).ready(function () {
         ]);
         self.authenticationType_in = ko.observableArray([
             { "text": "NO_AUTH", value: "NO_AUTH" },
-            { "text": "BASIC_AUTH", value: "BASIC_AUTH" }
+            { "text": "BASIC_AUTH", value: "BASIC_AUTH" },
+            { "text": "BASIC_AUTH Jenkins CSRF Protection (crumb)", value: "BASIC_AUTH_JENKINS_CSRF" }
```

Comment:
Should maybe be BASIC_AUTH_JENKINS_SECURITY? Since in jenkins it will only say security enabled true/false. If you dig into jenkins then you will find out that it means that it protects against CSRF.

Reply:
It is named CSRF Protection in Jenkins, and is one of many optiuons under Security.

Reply:
Oh yeah it was its own config under jenkins security, i thought it was always when you have security enabled.